packages/theme_tailor/example/test/json_test.dart

```diff
@@ -0,0 +1,23 @@
+import 'dart:convert';
+
+import 'package:example/json_serializable_example.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'random_values.dart';
+
+void main() {
+  test('Serializable theme <- from JSON', () {
+    final barColor = randomColor();
+    final fooNumber = randomInt();
+
+    final themeStr =
+        '''{"bar_color":${barColor.value},"foo_number":$fooNumber}''';
```

Comment:
it's not multiline string so we can replace ````` with ```

Reply:
That's right, I will change it